airflow/www/static/css/dags.css

```
@@ -31,10 +31,12 @@
 .dags-table-body {
   margin: 0 1px;
   overflow-x: auto;
+  padding-bottom: 34px;
+}
 
-  /* This allows small dropdowns to overlap the table element */
-  padding-bottom: 55px;
-  margin-bottom: -55px;
+.no-padding {
```

This class name suggests `padding: 0;` but it is only setting x-axis.
```suggestion
.no-x-padding {
```